pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1479,7 +1480,8 @@
         "3.9.0.0": "3.9.0",
         "4.0.0.0": "4.0.0",
         "4.0.10.0": "4.0.10",
-        "4.0.11.0": "4.0.11"
+        "4.0.11.0": "4.0.11",
+        "4.0.12.0": "4.3.0"
```

Comment:
@weshaggard The build throws error when adding 4.0.12.1: 4.3.1, asks to run /t:updatepackageindex target. Upon running that target, this is the change generated.